bin/node/runtime/src/lib.rs

```diff
@@ -1484,6 +1485,45 @@ impl pallet_assets::Config for Runtime {
 	type BenchmarkHelper = ();
 }
 
+impl pallet_assets::Config<Instance2> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type Balance = u128;
+	type AssetId = u32;
+	type Currency = Balances;
+	type CreateOrigin = AsEnsureOriginWithArg<EnsureSigned<AccountId>>;
```

Comment:
This needs to be the DEX pallet, this instance is not for users to register assets.

Reply:
Agreed the kitchen sink should set a good example. Fixed.